esp-hal/src/uart.rs

```diff
@@ -611,6 +611,17 @@ where
         Ok(space)
     }
 
+    /// Flush the transmit buffer.
+    ///
+    /// This function blocks until all data in the TX FIFO has been
+    /// transmitted.
+    #[instability::unstable]
+    pub fn flush(&mut self) -> Result<(), TxError> {
```

Comment:
I don't understand why you moved this, what did you achieve by moving it above two private functions that aren't visible in the documentation?

Reply:
TBH, I wanted to have functions sorted in code base as well - but it might be too much 😅